enterprise/snowpark-metrics.json

```diff
@@ -398,12 +368,12 @@
         "content": "Fill in the `Function Name` variable to view Memory and CPU metrics for this function or procedure.",
         "mode": "markdown"
       },
-      "pluginVersion": "11.1.0-69950",
+      "pluginVersion": "9.5.13",
       "targets": [
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_GRAFANACLOUD-SNOWPARKTEST1-PROM}"
+            "uid": "prometheus"
```

Comment:
Just for my learning, is there any reason this line change is needed?

Reply:
prometheus is part of every grafana install no need to point it to a specific id, the normal uid is enough